base/MAPL_FlapCLI.F90

```diff
@@ -1,35 +1,31 @@
 #include "MAPL_ErrLog.h"
 #include "unused_dummy.H"
 
-module MAPL_FlapCapOptionsMod
+module MAPL_FlapCLIMod
    use MPI
    use ESMF
    use FLAP
    use MAPL_KeywordEnforcerMod
    use MAPL_ExceptionHandling
-   use MAPL_CapOptionsMod
-   use pflogger
    implicit none
    private
 
-   public :: MAPL_FlapCapOptions
+   public :: MAPL_FlapCLI
 
-   type, extends(MAPL_CapOptions) :: MAPL_FlapCapOptions
+   type :: MAPL_FlapCLI
      type(command_line_interface) :: cli_options
    contains
       procedure, nopass :: add_command_line_options
-      procedure :: parse_command_line_arguments
-      procedure :: set_esmf_logging_mode
-   end type MAPL_FlapCapOptions
+   end type MAPL_FlapCLI
 
-   interface MAPL_FlapCapOptions
-      module procedure new_FlapCapOptions
+   interface MAPL_FlapCLI
```

Comment:
Is it possible to keep the original constructor names (and thus backward compatibility)?

Reply:
It is probably not worthy to keep the interface but make the codes messy